language/stdlib/modules/Option.move

```diff
@@ -1,16 +1,10 @@
 address 0x1 {
```

Comment:
I noticed in the formatted doc that EOPTION_IS_SET and EOPTION_NOT_SET have exactly the same descriptions.  The descriptions may be correct ("The Option is in an invalid state for the operation attempted."), but it's not as helpful as it could be.

Reply:
Will make it better.